Tests/test_image.py

```diff
@@ -86,6 +86,29 @@ def test_sanity(self):
         # with pytest.raises(MemoryError):
         #   Image.new("L", (1000000, 1000000))
 
+    def test_init(self):
+        try:
+            # No formats
+            Image.init([])
+            assert len(Image.ID) == 0
+            for ext in (".jpg", ".png"):
+                with pytest.raises(UnidentifiedImageError):
+                    Image.open("Tests/images/hopper" + ext)
```

Comment:
```suggestion
                    with Image.open("Tests/images/hopper" + ext):
                        pass
```

Apply #4232 to this test. Another option is to use e.g. `assert im.mode == "RGB"` instead of `pass`.